packages/gatsby/src/commands/build-html.ts

```diff
@@ -95,22 +106,22 @@ const renderHTMLQueue = (
             .catch(reject)
         })
     )
-      .then(resolve)
+      .then(() => resolve())
```

Comment:
Why did you need to change this?

Reply:
Because typescript caught a potential bug. This code is intended to resolve<void>, but it was actually resolving with the value of the above promise's. Which isn't void.